src/content/article/kodi-21-0-omega-beta-3/index.md

```diff
@@ -0,0 +1,99 @@
+
+- **Linux**:
+  - Pipewire sink got a fixup for sample rate switching. With that updated code, the back-end server of Pipewire will switch the base sample rate whenever Kodi is run without additional other clients. This way, Kodi's audio will be output in the best possible and lossless way throughout the audio chain, including soundserver and audio device.
+
+- **macOS**:
+
+- **Windows**:
+  - Improved chroma upscaling when are used Pixel Shaders or Software render methods and also DXVA only in specific cases: AMD + old drivers + 10-bit SDR.
+  - Improved channel layout match in WASAPI when audio drivers not support same exact speakers’ layout. This is only relevant for PCM (not passthrough).
+  - Increased the number of video surfaces in DXVA H.264 decoding to prevent video corruption in some rare H.264 streams.
+
+**PVR**
+
+**Python**
+
+**Skinning**
+
+**UPnP**
+
+**Videos**
+- `.nomedia` files can now be used to exclude folders inside a TV show: https://github.com/xbmc/xbmc/pull/24330
+
+**Music**
+- Fix an issue with m3u playlists which didn't show art for the second and subsequent tracks: https://github.com/xbmc/xbmc/pull/24675.
+- Allow navigation from artists directly to songs, missing out albums and discs.  This is more like how Spotify et al. navigate: https://github.com/xbmc/xbmc/pull/24649, forum thread https://forum.kodi.tv/showthread.php?tid=375313.
+
+**Known Issues**
+
+You can see open issues on our tracker at https://github.com/xbmc/xbmc/issues. However, to call out a couple of specifics, please be aware of the following if you're planning on upgrading:
+
+- There are seek issues with multi-part MP4 files on Android (https://github.com/xbmc/xbmc/issues/23673).
+- We've had reports that the new splash screen looks like an OpenGL corruption; we know, it's not a problem, so don't panic. We'd be happy to have your artwork contributions if you'd like to work on an alternative. 
+- If remote sources are unavailable at startup, users are seeing significant delays while waiting (https://github.com/xbmc/xbmc/issues/23958).
+
+We continue to march forward to what we hope will be a successful v21 Omega release in the new year. 
+
+If you love to live on the bleeding edge, you can get Beta 3 from [here](https://kodi.tv/download). Select your platform of choice, and look in the "Prerelease" section. Again, though, please: expect some breakage, don't use this as your daily installation unless you know how to get things working again, and please share your experiences back with us so we can really get going on those bugs.
+
+There is always a great deal more changes/fixes/updates than what we can highlight in these reports, so as usual you can see a fuller list of what has been merged between v21 and v20 [here](https://github.com/xbmc/xbmc/compare/Nexus...xbmc:21.0b3-Omega). For a more targeted look at the changes made since v21 Beta 2, you can see those changes [here](https://github.com/xbmc/xbmc/compare/21.0b2-Omega...21.0b3-Omega).
```

Comment:
```suggestion
There is always a great deal more changes/fixes/updates than we can highlight in these reports, so as usual you can see a fuller list of what has been merged between v21 and v20 [here](https://github.com/xbmc/xbmc/compare/Nexus...xbmc:21.0b3-Omega). For a more targeted look at the changes made since v21 Beta 2, you can see those changes [here](https://github.com/xbmc/xbmc/compare/21.0b2-Omega...21.0b3-Omega).
```